ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/Job_management.java

```diff
@@ -0,0 +1,342 @@
+import com.spectralogic.ds3client.helpers.options.WriteJobOptions;
+import com.spectralogic.ds3client.models.*;
+import com.spectralogic.ds3client.models.bulk.Ds3Object;
+import com.spectralogic.ds3client.networking.FailedRequestException;
+import com.spectralogic.ds3client.serializer.XmlProcessingException;
+import com.spectralogic.ds3client.utils.ResourceUtils;
+import javafx.util.converter.DateTimeStringConverter;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URISyntaxException;
+import java.nio.channels.FileChannel;
+import java.nio.channels.SeekableByteChannel;
+import java.nio.file.*;
+import java.security.SignatureException;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
+
+import static com.spectralogic.ds3client.helpers.Ds3ClientHelpers.*;
+import static com.spectralogic.ds3client.integration.Util.*;
+import static org.hamcrest.CoreMatchers.*;
+import static org.junit.Assert.*;
+import static org.junit.Assume.assumeThat;
+
+public class Job_management {
+
+    private static final Logger LOG = LoggerFactory.getLogger(Job_management.class);
+
+    private static Ds3Client client;
+
+    @BeforeClass
+    public static void startup() {
+        client = fromEnv();
+    }
+
+    @AfterClass
+    public static void teardown() throws IOException {
+        client.close();
+    }
+
+    @Test //also in smoke test--
+    public void modifyJobPriority() throws IOException, SignatureException, XmlProcessingException {
+        final String bucketName = "test_modify_job_priority";
+        try {
+            client.putBucket(new PutBucketRequest(bucketName));
+
+            final List<Ds3Object> objects = new ArrayList<>();
+            final Ds3Object obj = new Ds3Object("test", 2);
+            objects.add(obj);
+
+            final WriteJobOptions jobOptions = WriteJobOptions.create().withPriority(Priority.LOW);
+
+            final Ds3ClientHelpers.Job job =
+                    wrap(client).startWriteJob(bucketName, objects, jobOptions);
+
+            client.modifyJobSpectraS3(new ModifyJobSpectraS3Request(job.getJobId())
+                    .withPriority(Priority.HIGH));
+
+            final GetJobSpectraS3Response response = client
+                    .getJobSpectraS3(new GetJobSpectraS3Request(job.getJobId()));
+
+            assertThat(response.getMasterObjectListResult().getPriority(), is(Priority.HIGH));
+
+        } finally {
+            deleteAllContents(client, bucketName);
+        }
+    }
+
+    @Test
+    public void modifyJobName() throws IOException, SignatureException, XmlProcessingException {
+        final String bucketName = "test_modify_job_name";
+        try {
+            client.putBucket(new PutBucketRequest(bucketName));
+
+            final List<Ds3Object> objects = new ArrayList<>();
```

Comment:
you can simplify lines 106-108 to:
final List<Ds3Object> objects = Lists.Lists.newArrayList(new Ds3Object("test", 2));